openedx/core/djangoapps/util/user_messages.py

```diff
@@ -0,0 +1,125 @@
+"""
+Support for request messages to be shown to the user.
```

Comment:
Could you enhance this comment to give a quick overview and make it more clear what this is all about?  Maybe mention:
1. How it works: Showing on the same request or next request using Django messages.
2. Intention that there is one place in the platform (may say where) that will display these user messages.